lib/senkyoshi/models/wikipage.rb

```diff
@@ -16,6 +18,12 @@ def canvas_conversion(course, resources)
             #{@body}
           }
         end
+        if @extendeddata
+          @body = %{
+            #{@body}
+            #{_extendeddata(@extendeddata)}
+          }
+        end
```

Comment:
I am just wondering since we have 2 if statement to set the body if we should pull them out into their own function set_body function instead of making this function keep getting longer?